### DISCUSSION TITLE
Relax and Aim Assist lazer mods count for profile maximum combo

### DISCUSSION THREAD

**Discussion by Motorjam** [ppy/osu#17319] [Q&A] [answered]
If you use Relax and Aim Assist together with Aim Assist set to the highest value, you can full combo beatmaps without moving the cursor or tapping. This allows one to achieve high combos very easily, and these scores count towards the "Maximum Combo" stat on the profile page.
Example can be seen here: https://osu.ppy.sh/users/10743013
My Maximum Combo on my profile is 7,517, however the highest combo I have ever gotten legitimately was 4,162 (https://osu.ppy.sh/scores/osu/3896121944)
The 7,517 came from a score I did for fun on Painters of the Tempest, which used the Relax and Aim Assist mods, among others. 
![image](https://user-images.githubusercontent.com/53126725/158952185-60c316be-69ff-4014-a9ee-67cdead6e1f6.png)
![image](https://user-images.githubusercontent.com/53126725/158952299-6d18c65b-1b59-4b03-8b0e-095ad6b2dd2f.png)

**Comment by peppy** [comment, marked as answer]
Seems like automation mods shouldn't count for max combo profile stat..? @smoogipoo thoughts?

**Reply by bdach** [reply]
I honestly think that aim assist and relax should just be full on incompatible. Because this is going to be a problem with just about everything else (pp, multiplayer...)

**Reply by smoogipoo** [reply]
I think both of the above suggestions are reasonable.

**Reply by peppy** [reply]
I prefer the incompatibility path for sure

**Reply by Sublimelime** [reply]
PR'd "make them incompatible" fix in #17331.

**Reply by integerrr** [reply]
Will you be able to amend/revert the mentioned profile stats to what it was before?

**Reply by peppy** [reply]
not sure it's worth the effort

**Reply by integerrr** [reply]
Im asking because i feel like this will turn into another twitter shitstorm™️ if someone on twitter found out this being a valid way to cheese certain profile stats

**Comment by NotGumballer91** [comment]
Instead of automation mods, Maybe Fun mods shouldn't count for profile stats. It's just for "fun" after all. Just a thought.
